images/hook/entrypoint.sh

```diff
@@ -91,6 +91,7 @@ if [ $1 = "update" ]; then
 	kubectl --namespace monitoring patch alertmanagers.monitoring.coreos.com main --type=json -p='[{"op": "replace", "path": "/spec/replicas", "value": 2}]'
     fi
     # check for readiness of prometheus pod
+    timeout 5m sh -c "while ! kubectl get pod prometheus-k8s-0; do sleep 10; done"
```

Comment:
5 minutes seems like a long time for a pod to show up in the control plane -- leading to delayed failures if/when things go wrong.  Could we tune this down to fail fast?  E.g. previously it was 0s, how about 15s or 1m?

Same concern applies to the install hook.